webapp/blueprint.py

```diff
@@ -31,6 +32,11 @@ def root():
         return flask.redirect(JAAS_URL)
 
 
+@gui.route("/docs/<path:path>")
+def docs(path=""):
+    return flask.redirect(urllib.parse.urljoin("https://docs.jujucharms.com", path))
```

Comment:
will this hit any other redirects that help with the version to non-versioned urls as well?